Source/WebCore/dom/mac/ImageControlsMac.cpp

```diff
@@ -105,6 +106,7 @@ void createImageControls(HTMLElement& element)
     auto button = HTMLButtonElement::create(HTMLNames::buttonTag, element.document(), nullptr);
     button->setIdAttribute(imageControlsButtonIdentifier());
     controlLayer->appendChild(button);
+    controlLayer->setPseudo(ShadowPseudoIds::appleAttachmentControlsContainer());
```

Comment:
Why not name this something like `-apple-attachment-controls` or `-webkit-image-controls`?  This doesn't seem like it's specific to `<attachment>` 🤔

Reply:
It's used for `attachment` and attachment-backed `img` elements, so I think the `-attachment-` part of the pseudo element name is appropriate.

Reply:
The plan is to rename the image stuff to attachment anyways because it also covers PDF.